src/lib/stellar/horizonQueries.js

```diff
@@ -1,8 +1,12 @@
 import { error } from '@sveltejs/kit'
-import { Server, TransactionBuilder, Networks, StrKey, Asset } from 'stellar-sdk'
+import { TransactionBuilder, Networks, StrKey, Asset } from '@stellar/stellar-sdk';
+import { rpc } from '@stellar/stellar-sdk';
+
+
+const { Server } = rpc;
+const horizonUrl = 'https://horizon-testnet.stellar.org';
+const server = new Server(horizonUrl);
```

Comment:
Instead of importing `rpc` from the SDK, we'll want to import `Horizon` (you can add that into the line 2 imports), and then line 8 can just become `const server = new Horizon.Server(horizonUrl)`